taxi/views.py

```diff
@@ -28,6 +31,22 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def car_assign(request, pk: int):
+    user = request.user
+    car = Car.objects.get(id=pk)
+    car.drivers.add(user)
+    return redirect("taxi:car-detail", pk=pk)
+
+
+@login_required
+def car_unassign(request, pk: int):
+    user = request.user
+    car = Car.objects.get(id=pk)
+    car.drivers.remove(user)
+    return redirect("taxi:car-detail", pk=pk)
```

Comment:
You can make that as one function and change according template as it was in video. Not obligatory, good job, but think about it.